openIMIS/signal_binding/apps.py

```diff
@@ -0,0 +1,51 @@
+import io
+import json
+import os
+
+from django.apps import AppConfig
+import logging
+from django.apps import AppConfig
+
+
+logger = logging.getLogger(__name__)
+
+
+class SignalBindingConfig(AppConfig):
+    name = 'signal_binding'
+
+    def ready(self):
+        self.bind_service_signals()
+
+    def bind_service_signals(self):
+        def extract_app(module):
+            return "%s" % (module["name"])
+
+        def openimis_apps():
+            OPENIMIS_CONF = load_openimis_conf()
+            return [*map(extract_app, OPENIMIS_CONF["modules"])]
+
+        def load_openimis_conf(conf_file_param='../openimis.json'):
```

Comment:
Why are you duplicating the code from config/settings ? 
you can import from another module
https://github.com/openimis/openimis-be_py/blob/develop/modules-requirements.py#L10

Reply:
I made OPENIMIS_APPS settings variable and use import.